app/models/intake.rb

```diff
@@ -284,6 +284,8 @@ def self.searchable_fields
       self.email_domain = email_address.split('@').last.downcase
       self.canonical_email_address = compute_canonical_email_address
     end
+    self.primary_last_four_ssn = primary_ssn&.last(4) if encrypted_primary_ssn_changed?
```

Comment:
hmm... i think because of the fact that we don't expect it to be an action anyone but an engineer or console access would be able to do AND we never worried about keeping SSNs in sync before, it seems like adding complexity for a use case that hasn't ever been a problem so I'd probably hold off on it. I can't see a legitimate reason for anyone doing this in good faith.